test/com/facebook/buck/swift/testdata/swift_test/dep1/dep1.h

```diff
@@ -0,0 +1,19 @@
+//
+//  dep1.h
+//  dep1
+//
+//  Created by Tho Nguyen on 13/09/2016.
+//  Copyright © 2016 Test. All rights reserved.
+//
```

Comment:
I deleted both `Info.plist` and `dep1.h` since they were generated by `xcode`. I should have not committed those files.